src/View/StringTemplate.php

```diff
@@ -323,4 +324,65 @@ protected function _formatAttribute($key, $value, $escape = true)
 
         return $key . '="' . ($escape ? h($value) : $value) . '"';
     }
+
+    /**
+     * Adds a class and returns a unique list either in array or space separated
+     *
+     * ### Options
+     *
+     * - `useIndex` if you are inputting an array with an 'element' called 'class' and want the same returning
+     *                  you should set this to 'class'
+     * - `returnArray` Setting this to `false` will return a space separated string (default is `true`)
+     *
+     * @param array|string $input The array or string to add the class to
+     * @param array|string $newClass the new class or classes to add
+     * @param array $options See above for options
+     * @return array|string
+     */
+    public function addClass($input, $newClass, $options = [])
```

Comment:
For reference [OptionsAwareTrait](https://github.com/FriendsOfCake/bootstrap-ui/blob/master/src/View/Helper/OptionsAwareTrait.php) contains tried and tested code to add new class to existing list.